site/docs/components/combo-box/usage.mdx

```diff
@@ -25,23 +24,33 @@ data:
 - To select multiple options from fewer than five. Instead, use [`CheckboxGroup`](../checkbox).
 - To make an instantaneous selection between two binary choices, for example “on” and “off”. Instead, use [`Switch`](../switch).
 
-### Import
+#### Controlled/Uncontrolled
+
+Just like native input elements in React, Combo Box can be controlled or uncontrolled. By default Combo Box is uncontrolled. Use the `defaultValue` and `defaultSelected` props to control the default value of Combo Box.
+If you want to control the value of Combo Box, use the `value` and `selected` props. The `onSelectionChange` prop is required when using the controlled approach.
 
-{/* Update the text and code snippet below as needed: */}
+### Import
 
 To import `ComboBoxNext` from the lab Salt package, use:
 
 ```js
-import { ComboBoxNext } from "@salt-ds/lab";
+import { ComboBoxNext, Option, OptionGroup } from "@salt-ds/lab";
```

Comment:
Is OptionList missing here?

Reply:
I'm trying to keep OptionList private for now